internal/provider/resource_docker_network.go

```diff
@@ -155,51 +151,3 @@ func resourceDockerNetwork() *schema.Resource {
 		},
 	}
 }
-
```

Comment:
Could you re-add the deleted functions or is there a reason you deleted them?

Reply:
Ahh yes sorry I can. I deleted it because of my own OCD when the linter yells at my for "unused" functions 😅

Reply:
Done!